### DISCUSSION TITLE
cmi merchant

### DISCUSSION THREAD

**Discussion by noxidevad** [Zrips/CMI#5539] [Feature Requests]
As from my question - https://github.com/Zrips/CMI/issues/5532

I would like to request a new feature to be introduced within CMI. that involves using /cmi merchant

Ability to create potential custom villagers (that can sell exactly what we wish them to sell, ie custom items such as Slimefun)
Ability 
Ability to edit the merchant gui (in-game potentially) kinda like your gui's already inside cmi like the kit editor
Ability to lock trades so users do not lose trades within that particular villager
Ability to have an admin type villager so 

Future perhaps
Ability for players to create there own villager shops.

Hopefully managed to explain myself well,

**Comment by CloudeLecaw** [comment]
Sounds like the plugin shopkeepers :)

All togehter ubasked for a whole plugin with many features. I dont think it will be added because its a bit to "farnout of the window" an furthermore a whole new section. Personaly i also would love to see a feature like that. What u requested would be a nice addon for many servers ^.^

 Well we will see. Like allways zrips has to decide :)
